Dockerfile

```diff
@@ -5,6 +5,9 @@ FROM almalinux:10-minimal@sha256:9cdcbb8b9b0e522e4c190d4f1db67ebd443736630ad77cf
 ARG S6_OVERLAY_VERSION=3.2.1.0
 # renovate: datasource=github-releases depName=apollographql/router
 ARG APOLLO_ROUTER_VERSION=2.3.0
+# Preview router version - set this to build preview images automatically
+# once new preview is out, we'll uncomment this.
+#ARG APOLLO_ROUTER_PREVIEW_VERSION=2.3.0-preview.0
```

Comment:
I'm tempted to ignore this for now, and just focus on getting on-demand builds of router previews. However, I'm writing a doc at the moment about publishing multiple router versions etc, and I'll include a section on preview versions. We can probably rig up renovate so that it will bump this for us, as and when preview versions exist but if we can get the on-demand part working for now that would be a great start.